pkg/server/service.go

```diff
@@ -268,3 +276,21 @@ func (c *criContainerdService) getDeviceUUID(path string) (string, error) {
 func imageFSPath(rootDir, snapshotter string) string {
 	return filepath.Join(rootDir, fmt.Sprintf("%s.%s", plugin.SnapshotPlugin, snapshotter))
 }
+
+// startProfilingServer start http server to profiling via web interface
+func startProfilingServer(addr string, port string) {
+	if ip := net.ParseIP(addr); ip != nil {
+		endpoint := net.JoinHostPort(addr, port)
+		mux := http.NewServeMux()
+		mux.HandleFunc("/debug/pprof/", pprof.Index)
+		mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+		mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+		mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+		err := http.ListenAndServe(endpoint, mux)
+		if err != nil {
+			glog.Errorf("Failed to start server: %v", err)
```

Comment:
/s/server/profiling server/

Reply:
thanks a lot for your review. I will do as your suggestion.

Reply:
Done